controllers/past-grants/views/recipient-detail.njk

```diff
@@ -0,0 +1,116 @@
+                                <dt>Charity number</dt>
+                                <dd>{{ organisation.charityNumber }}</dd>
+                            {% endif %}
+
+                            {% if organisation.companyNumber %}
+                                <dt>Company number</dt>
+                                <dd>{{ organisation.companyNumber }}</dd>
+                            {% endif %}
+
+                            <dt>Location</dt>
+                            <dd>
+                                {% if recipientLocalAuthorities.length === 1 %}
+                                    {% set grant = recipientGrants[0] %}
+                                    {{ buildLocationList(grant.beneficiaryLocation) }}
+                                {% else %}
+                                    Multiple locations
+                                {% endif %}
+                            </dd>
+
+                        </dl>
+
+                        <p class="u-margin-top-l">
+                            {% set orgNameSafe = organisation.name | urlencode %}
+                            <strong>
+                                <a href="https://www.google.co.uk/search?q={{ orgNameSafe }}">
+                                    Search for {{ organisation.name }} with Google
+                                </a>
+                            </strong>
+                        </p>
+
+                    </div>
+                </div>
+
+                <div class="content-sidebar__secondary u-tone-background-tint u-padded">
+                    <h2>Awarding programmes</h2>
+                        <ul>
+                            {% for programme in recipientProgrammes %}
+                                <li class="u-margin-bottom t5">
+                                    <a class="accent--pink a--text"
+                                       href="{{ pgLink('programme', programme.label) }}">
+                                        {{ programme.label }}</a>
+                                    {% if recipientProgrammes.length > 1 %}
+                                        <br />
+                                        <small>
+                                            ({{ programme.count }} award{{ programme.count | pluralise("", "s") }})
+                                        </small>
+                                    {% endif %}
+                                </li>
+                            {% endfor %}
+                        </ul>
+                </div>
+
+            </div>
+        </div>
+
+        <div class="u-inner-wide-only">
+            <h2>{{ totalResults }} projects funded by Big Lottery Fund (£{{ totalAwarded }} awarded in total)</h2>
+            <div class="flex-grid">
+                {% for grant in recipientGrants %}
+                    <div class="flex-grid__item">
+                        <div class="content-box content-box--borderless u-tone-background-pale-grey u-block-full u-no-margin">
```

Comment:
This feels like accidental styling, i.e. chaining a bunch of classes until we get what we want. Is there a pattern for this already, if not should there be?

Reply:
Good call, the action card does this. Have a look now!